File: OnlinePK-Android/lib-user/src/main/java/com/netease/yunxin/nertc/demo/user/CommonUserNotify.java
/*
 * Copyright (c) 2021 NetEase, Inc.  All rights reserved.
 * Use of this source code is governed by a MIT license that can be found in the LICENSE file.
 */

package com.netease.yunxin.nertc.demo.user;

/**
 * Created by luc on 2020/11/16.
 */
public abstract class CommonUserNotify implements UserCenterServiceNotify {

    @Override
    public void onUserLogin(boolean success, int code) {

    }

    @Override
    public void onUserLogout(boolean success, int code) {

    }

    @Override
    public void onError(Throwable exception) {

    }

    @Override
    public void onUserInfoUpdate(UserModel model) {

    }
}

File: OnlinePK-Android/lib-user/src/main/java/com/netease/yunxin/nertc/demo/user/CommonUserNotify.kt
/*
 *  Copyright (c) 2021 NetEase, Inc.  All rights reserved.
 *  Use of this source code is governed by a MIT license that can be found in the LICENSE file
 */

package com.netease.yunxin.nertc.demo.user

/**
 * Created by luc on 2020/11/16.
 */
abstract class CommonUserNotify : UserCenterServiceNotify {
    override fun onUserLogin(success: Boolean, code: Int) {}
    override fun onUserLogout(success: Boolean, code: Int) {}
    override fun onError(exception: Throwable?) {}
    override fun onUserInfoUpdate(model: UserModel?) {}
}